marc.go

```diff
@@ -370,27 +348,36 @@ func TranslateLanguageCodes(recordCodes []string, languageCodes map[string]strin
 }
 
 // getLinks take a MARC record and eturns an array of Link objects from the 856 field data.
-func getLinks(marcrecord *marc21.Record) []Link {
+func getLinks(fmlRecord fml.Record) []Link {
 	var links []Link
-	marc856 := marcrecord.GetFields("856")
+	marc856 := fmlRecord.DataField("856")
```

Comment:
I think you might be able to do something like this here to avoid some of the more complicated indicator checking down below:
```
marc856 := fmlRecord.Filter("856|40|3uyz", "856|41|3uyz")
```

Reply:
I guess this won't work if you expect any of those subfields to be missing.

Reply:
I believe the Filter method does not return the structure necessary to use the data the way we do in Links.

DataFields Example:
```go
([]fml.DataField) (len=1 cap=1) {
 (fml.DataField) {
  Indicator1: (string) (len=1) "4",
  Indicator2: (string) (len=1) "1",
  Tag: (string) (len=3) "856",
  SubFields: ([]fml.SubField) (len=1 cap=1) {
   (fml.SubField) {
    Code: (string) (len=1) "u",
    Value: (string) (len=130) "https://www.cambridge.org/core/journals/proceedings-of-the-international-astronomical-union/issue/024C8F71836A32451299A8FFC4BAC933"
   }
  }
 }
}
```

Filter example:
```go
([][]string) (len=1 cap=1) {
 ([]string) (len=1 cap=1) {
  (string) (len=130) "https://www.cambridge.org/core/journals/proceedings-of-the-international-astronomical-union/issue/024C8F71836A32451299A8FFC4BAC933"
 }
}
```

Reply:
Basically, I need to know which subfield the data is from to map it into our Link struct.

Reply:
If the subfields will always be present then they will be returned in the order in which you request them in your filter query. If some of the subfields may sometimes be missing, then yeah it won't work.